generators/service-appid/templates/swift/instrumentation.swift

```diff
@@ -27,4 +27,5 @@ func initializeServiceAppid() throws {
     kituraCredentials = Credentials()
     kituraCredentials.register(plugin: webappKituraCredentialsPlugin)
     Log.info("Found and loaded credentials for AppID.")
+    return kituraCredentials
```

Comment:
Should we also export the `webappKituraCredentialsPlugin` so the user can access it? Or are there no useful APIs on this object that a user might need?

Reply:
done